### PR TITLE
Form control cleanup.

### PR DESCRIPTION
**Pull request by bernalrs**

### CODE REVIEW

**Comment by bernalrs** [comment]
@InsidersByte  Ready to merge.

File: src/MaterialUI.re [hunk 1]
```diff
@@ -565,16 +565,74 @@ module Drawer = {
     );
 };
 
-module FormControl = {
```
Comment (InsidersByte) [inline src/MaterialUI.re]: did you mean to delete this module?

File: src/MaterialUI.re [hunk 2]
```diff
@@ -565,16 +565,74 @@ module Drawer = {
...
+        }
+      ),
+    children
+  );
+
+module FormControlLabel = {
+  [@bs.module "material-ui/Form"] external reactClass : ReasonReact.reactClass =
+    "FormControlLabel";
   let make =
+      /* technically a union bool|string but why are you passing a string to `checked`? */
```
Comment (InsidersByte) [inline src/MaterialUI.re]: move this to above the checked argument

**Review by InsidersByte — CHANGES_REQUESTED** [review]

File: src/MaterialUI.re (hunk 1 above)
Comment (bernalrs) [reply src/MaterialUI.re]: Actually no, not sure how that got removed.

**Comment by bernalrs** [comment]
@InsidersByte done.

**Review by InsidersByte — APPROVED** [review]

**Pull request merged**